bsp-grunt.js

```diff
@@ -136,23 +135,23 @@ module.exports = function(grunt, config) {
       }
     },
 
-    requirejs: {
-      dynamic: {
-        options: {
-          baseUrl: '<%= bsp.scripts.devDir %>',
-          dir: '<%= bsp.scripts.minDir %>',
-          modules: '<%= bsp.scripts.rjsModules %>',
-          optimize: 'uglify2'
-        }
-      }
-    },
-
-
     watch: {
       less: {
         files: '<%= bsp.styles.srcDir %>' + '/**/*.less',
         tasks: ['bsp-config-dest', 'copy:styles', 'less:compile', 'copy:compiledCSSForWatcher']
       }
+    },
+
+    systemjs: {
+      dist: {
+        options: {
+          configFile: '<%= bsp.scripts.devDir %>/config.js',
+          configOverrides: grunt.config('bsp.systemjs')
+        },
+        files: [
+          { '<%= bsp.scripts.minDir %>/main.min.js': '<%= bsp.scripts.devDir %>/main.js' }
```

Comment:
Instead of hardcoding to main.js, can we support a default of main.js, but then if in gruntfile.js a different file name or an array of file is specified, we use that instead?